apps/test-units/src/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include "units.h"
+#include <cmath>
+
+// These are various namespaces defined that make units esaier to work with
+using namespace units::length;
+using namespace units::force;
+using namespace units::current;
+using namespace units::angular_velocity;
+using namespace units::literals;
+using namespace units::angle;
+using namespace units::time;
+using namespace units::current;
+using namespace units::impedance;
+using namespace units::voltage;
+using namespace units::mass;
+
+int main() {
+
+    // Testing angular velocity in degrees per second and angle in degrees
+
+    degrees_per_second_t speed = degree_t(5)/second_t(1.0); // creates an angular velocity type object with velocity of 5 degrees/second
+
+    // Multiplying 5 degrees/sec by 5 seconds yields 25 degrees
+    // Not explicitly declaring it as degree_t would assume it as radians, which is the SI unit
+    degree_t degreesMoved = speed * second_t(5); 
+
+    // Printing both the angular velocity and the degrees subtended: 
+    std::cout << speed << std::endl;
```

Comment:
Can you replace all `std::cout <<` with `Utility::logger <<`? You'll need to include the Logger library